pkg/rancher-desktop/components/UpdateStatus.vue

```diff
@@ -178,7 +178,7 @@ class UpdateStatus extends UpdateStatusProps {
     if (typeof markdown !== 'string') {
       return undefined;
     }
-    const unsanitized = marked(markdown);
+    const unsanitized = marked(markdown) as string;
```

Comment:
We should have a comment here describing why this is safe. (In this case, we're not using any extensions.)

This is mostly because if we violated that in the future, we have a record of what might have broken.